c7n/resources/account.py

```diff
@@ -970,3 +970,70 @@ def process(self, resources):
                 if e.response['Error']['Code'] == 'ResourceNotFoundException':
                     return
                 raise
+
+
+@filters.register('xray-encrypt-key')
+class XrayEncrypted(Filter):
+    """Determine if xray is encrypted.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: xray-encrypt-with-default
+                resource: aws.account
+                filters:
+                  - type: xray-encrypt-key
+                    key: default
+              - name: xray-encrypt-with-kms
+                  - type: xray-encrypt-key
+                    key: kms
+    """
+
+    permissions = ('xray:GetEncryptionConfig',)
+    schema = type_schema(
+        'xray-encrypt-key',
+        required=['key'],
+        key={'type': 'string', 'enum': ['default', 'kms']}
```

Comment:
organizations also like to verify the actual key in use (via alias specification), versus just presence of kms, and in this context 'default' seems rather ambiguous that refers to no encryption.

Reply:
I will do a "one of" fix for alias and also change 'default' to 'absent' for encryption.

Reply:
this didn't seem to address the other part of the comment, namely users are typically looking to set/filter on a particular kms key, not merely the presence of kms.

Reply:
My plan is to support the two kinds of test - one for symbolic tests (default or kms) and the second for a specific key (supporting alias and arn). I believe this covers all the cases you wanted.

Reply:
it does, thanks